tests/scripts/ci_e2e.sh

```diff
@@ -67,9 +67,9 @@ install_pytest_requirements
 # Pytest is not able to have both --timeout & --workers, so do not add --timeout or --workers in the shell script
 if [[ -n "${TEST_TIMEOUT:-}" ]]; then
   
-  timeout  "${TEST_TIMEOUT}" pytest --host ${MILVUS_SERVICE_NAME} --port ${MILVUS_SERVICE_PORT} \
+  timeout  "${TEST_TIMEOUT}" pytest ./testcases --host ${MILVUS_SERVICE_NAME} --port ${MILVUS_SERVICE_PORT} \
```

Comment:
@Bennu-Li 
Please help review!
I changed the root dir for pytest in this script.